lib/acts_as_list/active_record/acts/scope_method_definer.rb

```diff
@@ -68,7 +86,10 @@ def self.idify(caller_class, name)
     return name if name.to_s =~ /_id$/
 
     if caller_class.reflections.key?(name.to_s)
-      caller_class.reflections[name.to_s].foreign_key.to_sym
+      foreign_key = caller_class.reflections[name.to_s].foreign_key
+      # Handle composite foreign keys (Arrays) by returning as-is
+      # Single foreign keys should be converted to symbols
+      foreign_key.is_a?(Array) ? foreign_key : foreign_key.to_sym
```

Comment:
Just wondering here. We convert the columns to symbols for single foreign keys but not for the composite ones? I'm not sure why we do this anyway but maybe those should be converted also?